packages/codemods/src/modRunner/runnerUtilities.ts

```diff
@@ -72,18 +71,18 @@ export function loadMod(path: string, errorCallback: (e: Error) => void): Maybe<
   return Nothing();
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function filterMods(codeMods: CodeMod<any>[], semverRange: Range) {
-  return codeMods.filter(mod => shouldRunMod(mod, semverRange));
+export function getEnabledMods(getPaths = getModsPaths, loadM = loadMod) {
+  return getPaths()
+    .map(pth => {
+      console.log('fetching codeMod at ', pth);
+      return loadM(pth, e => {
+        console.error(e);
```

Comment:
This is a weird signature. Any reason not to throw if fetching fails? Or have an `Either` type instead of a `Maybe`?

Reply:
I'll add either in another pr along with monads.